packages/guui/grid.js

```diff
@@ -113,25 +113,47 @@ const normaliseProps = (props: BreakpointProps | number): BreakpointProps => {
     return props;
 };
 
-export const Row = ({ children }: { children: React.Node }) => (
-    <RowStyled>{children}</RowStyled>
-);
+export const Row = ({
+    htmlTag,
+    children,
+}: {
+    htmlTag?: string,
+    children: React.Node,
+}) => {
+    const GridRow = RowStyled.withComponent(htmlTag);
+
+    return <GridRow>{children}</GridRow>;
+};
+
+Row.defaultProps = {
+    htmlTag: 'div',
+};
```

Comment:
Interesting thing that was flagged by `eslint` was originally I didn't use `defaultProps` but instead I assigned a default value to `htmlTag`.  Eslint reported a`react/require-default-props` error when doing this. Further investigation lead to me to find: https://github.com/yannickcr/eslint-plugin-react/issues/1009

> default values are re-evaluated on every invocation of the function. defaultProps, on the other hand, are evaluated once and cached at the module level - across every invocation of the function. Even if React does nothing with regard to them (which I doubt, but haven't bothered to dig up evidence of), defaultProps are more performant than default values.

Reply:
So interesting! Nice one